src/main/java/org/fungover/haze/Main.java

```diff
@@ -18,56 +18,87 @@ public class Main {
     static Logger logger = LogManager.getLogger(Main.class);
 
     public static void main(String[] args) {
-        Initialize initialize = new Initialize();
-        initialize.importCliOptions(args);
+        Initialize initialize = getInitialize(args);
```

Comment:
I suggest to move getInitialize method to our Initialize class as a factory method. Doesn't belong in Main class